benchmark/run_csharp_benchmark.sh

```diff
@@ -12,7 +12,7 @@ DATA_PATH=$9
 NUM_ITERATION=${10}
 IS_SQL=${11}
 
-for i in {1..22}
+for i in {1..22} 1a 8a 1ha 8ha
```

Comment:
I would make this optional (you can add comments and add these, for example).

Reply:
Done